syncopy/io/read_external.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
As discussed, I would prefer for such distinct data as `.spy`, `.mat` and whatever the `nwb` extension actually is to have also distrinct readers. I think for the user it's fair to say `load_spy` if you want to load Syncopy data, `load_ft` if you want to to load FT data and `load_nwb` for NWB

Reply:
Deleted the global read manager and renamed `read_nwb` to `load_nwb` to keep naming consistent, cf PR commit